cloud-nio/cloud-nio-impl-drs/src/test/scala/cloud/nio/impl/drs/DrsCloudNioFileProviderSpec.scala

```diff
@@ -39,6 +39,8 @@ class DrsCloudNioFileProviderSpec extends AnyFlatSpecLike with Matchers with Moc
     fileSystemProvider.getScheme should be("drs")
     fileSystemProvider.getHost("drs://dg.123/abc") should be("dg.123")
     fileSystemProvider.getHost("drs://dg.example.com/abc") should be("dg.example.com")
+    fileSystemProvider.getHost("drs://dg.4503:dg.4503/abc") should be("dg.4503")
+    fileSystemProvider.getHost("drs://dg.4503:abc") should be("dg.4503")
```

Comment:
Are there more known-not-made-up-and-in-use [compact identifier-based](https://ga4gh.github.io/data-repository-service-schemas/preview/develop/docs/#_calling_meta_resolver_apis_for_compact_identifier_based_drs_uris) (CIB) URIs that can be regression tested?

Reply:
I created these based on the examples in https://broadworkbench.atlassian.net/browse/WA-331.  Please suggest additional ones if I missed them.